services/harmony/app/middleware/earthdata-login-token-authorizer.ts

```diff
@@ -1,10 +1,33 @@
 import { RequestHandler } from 'express';
+import { LRUCache } from 'lru-cache';
 
 import HarmonyRequest from '../models/harmony-request';
 import { getUserIdRequest } from '../util/edl-api';
+import env from '../util/env';
 
 const BEARER_TOKEN_REGEX = new RegExp('^Bearer ([-a-zA-Z0-9._~+/]+)$', 'i');
 
+/**
+ * Wrapper function of getUserIdRequest to be set to fetchMethod of LRUCache.
+ *
+ * @param token - EDL bearer token
+ * @param _sv - default value parameter of LRUCache fetchMethod, unused here
```

Comment:
nit: technically `_sv` is the "stale value", not the default value. I guess there are some use cases where knowing what the old value was is necessary to get the new value (like maybe a counter or something), but not in our case.